src/goPackages.ts

```diff
@@ -75,17 +81,21 @@ function gopkgs(workDir?: string): Promise<Map<string, string>> {
 					}
 					const index = pkgPath.lastIndexOf('/');
 					const pkgName = index === -1 ? pkgPath : pkgPath.substr(index + 1);
-					pkgs.set(pkgPath, pkgName);
+					pkgs.set(pkgPath, {
+						name: pkgName,
+						isStd: pkgPath.includes(process.env['GOROOT'])
```

Comment:
perhaps .startsWith is a better function to use here. We expect the path to start with the goroot right?

```suggestion
						isStd: pkgPath.startsWith(process.env['GOROOT'])
```

Also, do we want to take care of the case where `process.env['GOROOT']` doesn't have any value set? It shouldn't happen so, but if it does, do we might want to default to false here right?